pr-builder.rst

```diff
@@ -0,0 +1,93 @@
+.. post:: July 14, 2019
+   :tags: gsoc, pr-builder, continuous-documentation, feature
+   :author: Saadmk11
+   :location: DHA
+
+Autobuild docs for Pull Requests
```

Comment:
We typically use titlecase for blogs and I'd spell out documentation or use something like "Building the Docs".

"Autobuild docs for Pull Requests" -> "Building the Docs in Pull Requests" or "Autobuild Documentation in Pull Requests".

Reply:
Autobuild Documentation in Pull Requests sounds nice. Thanks :)